ui-tests/src/test/java/io/syndesis/qe/fragments/common/form/Form.java

```diff
@@ -267,7 +267,7 @@ public void fillByLabel(Map<String, String> data) {
     }
 
     public static void waitForInputs(int timeInSeconds) {
-        $(By.cssSelector(".form-control")).waitUntil(exist, timeInSeconds * 1000);
+        $(By.cssSelector("form[data-testid=\"connector-configuration-form\"]")).waitUntil(exist, timeInSeconds * 1000);
```

Comment:
This will not be working in other forms. For example, calling `fill in values by element data-testid` in the WebHook step failed because the form doesn't have `connector-configuration-form` testid. Maybe just for waiting, the waiting for any Form which contains input or select is enough. E.g. `By.xpath("//form[.//input or .//select]")`

Reply:
Good job on pointing that out, you are right... let's change it to your suggestion and see if that's enough for all forms